aws/logs_monitoring/README.md

```diff
@@ -48,12 +48,46 @@ If you had previously enabled your AWS Integration using the [following CloudFor
 [101]: https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/#set-up-triggers
 [102]: https://github.com/DataDog/cloudformation-template/tree/master/aws
 
+{{% /tab %}}
+{{% tab "Multi-Region & Multi-Account" %}}
+
+### Multi-Region & Multi-Account
+
+Use CloudFormation StackSets to deploy the Forwarder across multiple AWS regions and/or accounts automatically.
+
+#### Single account, multiple regions
+
+1. Deploy the Forwarder template using [CloudFormation StackSets][151] in the AWS Console.
+2. In the **Permissions** section, optionally provide an IAM admin role ARN, or leave the default IAM execution role name (`AWSCloudFormationStackSetExecutionRole`).
```

Comment:
```suggestion
2. In the **Permissions** section, leave **Service-managed permissions** selected to use the default IAM execution role name (`AWSCloudFormationStackSetExecutionRole`), or optionally select **Self-service permissions** and provide an IAM admin role ARN .
```

Reply:
Thanks for clarifying. I've removed the suggestion here, and added a couple of earlier suggestions to clarify that this is for AWS Organizations.